src/commands/deploy/legacy.ts

```diff
@@ -220,6 +223,43 @@ const promptForEnvFields = async (list: string[]) => {
   return answers;
 };
 
+async function canUseZeroConfig(cwd: string): Promise<boolean> {
+  try {
+    const pkg = (await readPackage(join(cwd, 'package.json'))) as null | Error | Package;
```

Comment:
Why are you casting the returned value?

Reply:
@styfle because by default `readPackage` returns a different type of `Package`, which seems to not be equal to `package.json`

Reply:
Actually, let me adjust this, since it is just one property that is optional

Reply:
@styfle done